dulwich/client.py

```diff
@@ -46,6 +46,8 @@
 import subprocess
 import sys
 
+WIN = sys.platform.startswith('win')
```

Comment:
please move this code to below, where it is actually used and please check for just 'win32' as that seems to be the only relevant value.